pykechain/models/widgets/widgets_manager.py

```diff
@@ -2213,6 +2206,37 @@ def add_scopemembers_widget(
 
         return widget
 
+    def add_project_info_widget(
+            self,
+            title: TITLE_TYPING = False,
+            parent_widget: Optional[Union[Widget, str]] = None,
+            **kwargs):
```

Comment:
```suggestion
            **kwargs) -> "Widget":
```
unsure how to pu tthe return type here; check with other.